README.md

```diff
@@ -18,7 +18,7 @@ To use the USPS APIs you must be a registered user.
 5. Additionally, you will need to have a Customer ID (CRID) and Mailer ID (MID)
 
 ## Using the APIs in the USPS Test Environment
-All USPS APIs listed below are Production URL.  To use the API in Test Environment you need to change the URL from https://api.usps.com to https://api-cat.usps.com.
+All USPS APIs listed below are Production URL.  To use the API in Test Environment you need to change the URL from https://apis.usps.com to https://apis-cat.usps.com.
```

Comment:
I was going to create a very similar PR to this one as I just learned from customer support that the current API URLs in the README are incorrect, but saw this PR open so i thought I'd instead just comment on here. One thing to note is that in addition from updating the endpoint from `api.usps.com` to `apis.usps.com`, I was informed that the testing endpoint is no longer `apis-cat.usps.com` but `apis-tem.usps.com`